houston/RobotStreamer.py

```diff
@@ -42,6 +42,11 @@ def __init__(self):
         videoPort = endpointData["port"]
         self.stream_key = os.environ.get("STREAMKEY")
 
+        # Send camera alive
+        robot_util.sendCameraAliveMessage(
+            self.api_server, self.camera_id, self.stream_key
+        )
+
```

Comment:
[12:49 PM] kkboglinz: check to see if your sendCameraAliveMessage function is working. also it should come before the ffmpeg startup